### PR TITLE
WIP: added unfinished no_std docs

### PR DESCRIPTION
**Pull request by bitwave**
Blocked by #281

### CODE REVIEW

File: docs/src/advanced_features/no_std/no_std.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+```
+
+Then build your project e.g. for `aarch64-unknown-none` using
+```
+cargo build --no-default-features --target aarch64-unknown-none
+```
+
+## Use custom timing
+
+TODO: add here how to implement custom timing function when [PR](https://github.com/AFLplusplus/LibAFL/pull/281) is ready.
```
Comment (domenukk) [inline docs/src/advanced_features/no_std/no_std.md:18]: @bitwave want to add a simple example?

**Comment by bitwave** [comment]
as a dumb example.

**Comment by domenukk** [comment]
Maybe make it a tiny bit less dumb? :D
Name it secs instead of ms, or maybe use nanos instead

**Comment by domenukk** [comment]
Thinking of it, shouldn't we expose nanos, instead of millis to extern?

**Pull request merged**